boot/boot.S

```diff
@@ -21,6 +22,26 @@ init_dl: .byte 0
 	lgdt gdtdesc
 
 	PRINT_STR $protecte_msg, $23
+
+	# The following sectors (2-13) are loaded in order to print
+	# the illuminatrix icon. The information of the interruptions
+	# and X,Y coordinates are stored in these sectors.
+	# TODO: Obviously this part has to be re-factorized.
+	LOAD_NSECTOR $2
```

Comment:
probably we need to find a better way to load more than one sector, please check better-next-sector-loading and load kernel fixed address.